website/content/en/docs/crds-scope.md

```diff
@@ -0,0 +1,94 @@
+---
+title: CRD scope with Operator SDK
+linkTitle: CRD Scope
+weight: 60
+---
+
+## Overview
+
+The CustomResourceDefinition (CRD) scope can also be changed for cluster-scoped operators so that there is only a single 
+instance (for a given name) of the CRD to manage across the cluster.
+
+> **NOTE**: Cluster-scoped CRDs are **NOT** supported with the Helm operator. While Helm releases can create 
+>cluster-scoped resources, Helm's design requires the release itself to be created in a specific namespace. Since the 
+>Helm operator uses a 1-to-1 mapping between a CR and a Helm release, Helm's namespace-scoped release requirement 
+>extends to Helm operator's namespace-scoped CR requirement.
+
+For each CRD that needs to be cluster-scoped, update its manifest to be cluster-scoped.
+
+* `deploy/crds/<full group>_<resource>_crd.yaml`
```

Comment:
@camilamacedo86 I can see that we're linking to this same CRD scope doc from both the old and new operator-scope docs.
However in context of the new doc for KB layout, these paths aren't valid anymore. We have `config/crd` instead of `deploy/crds`.
Same for `pkg/apis/<group>/<version>/<kind>_types.go`.

This section needs to be updated for the KB layout as well. 
So can you please update this and move it back into both the operator-scope.md docs.

Reply:
It shows out of scope. I did not write this doc since it is not really part of operator-scope. 
I just split the doc for we are able to link it.